Nos/Views/New Note/NewNoteView.swift

```diff
@@ -252,7 +252,11 @@ struct NewNoteView: View {
             } else {
                 try await relayService.publishToAll(event: jsonEvent, signingKey: keyPair, context: viewContext)
             }
-            analytics.published(note: jsonEvent)
+            if let replyToNote {
```

Comment:
```suggestion
            if replyToNote != nil {
```
fixes the warning I saw in Xcode.